invenio_communities/assets/semantic-ui/js/invenio_communities/search/components.js

```diff
@@ -0,0 +1,409 @@
+  return (
+    <aside aria-label="filters" id="search-filters">
+      <Toggle
+        title="Versions"
+        label="View all versions"
+        filterValue={["allversions", "true"]}
+      />
+      {aggs.map((agg) => {
+        return (
+          <div className="ui accordion" key={agg.title}>
+            <BucketAggregation title={agg.title} agg={agg} />
+          </div>
+        );
+      })}
+      <SearchHelpLinks />
+    </aside>
+  );
+};
+
+export const RDMBucketAggregationElement = ({ title, containerCmp }) => {
+  return (
+    <Card className="borderless-facet">
+      <Card.Content>
+        <Card.Header>{title}</Card.Header>
+      </Card.Content>
+      <Card.Content>{containerCmp}</Card.Content>
+    </Card>
+  );
+};
+
+export const RDMToggleComponent = ({
+  updateQueryFilters,
+  userSelectionFilters,
+  filterValue,
+  label,
+  title,
+  isChecked,
+}) => {
+  const _isChecked = (userSelectionFilters) => {
+    const isFilterActive =
+      userSelectionFilters.filter((filter) => filter[0] === filterValue[0])
+        .length > 0;
+    return isFilterActive;
+  };
+
+  const onToggleClicked = () => {
+    updateQueryFilters(filterValue);
+  };
+
+  var isChecked = _isChecked(userSelectionFilters);
+  return (
+    <Card className="borderless-facet">
+      <Card.Content>
+        <Card.Header>{title}</Card.Header>
+      </Card.Content>
+      <Card.Content>
+        <Checkbox
+          toggle
+          label={label}
+          name="versions-toggle"
+          id="versions-toggle"
+          onClick={onToggleClicked}
+          checked={isChecked}
+        />
+      </Card.Content>
+    </Card>
+  );
+};
+
+export const RDMCountComponent = ({ totalResults }) => {
+  return <Label>{totalResults.toLocaleString("en-US")}</Label>;
+};
+
+export const RDMEmptyResults = (props) => {
+  const queryString = props.queryString;
+  return (
+    <>
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="search" />
+          "No results found!"
+        </Header>
+        {queryString && (
+          <em>
+            "Current search" "{queryString}"
+          </em>
+        )}
+        <br />
+        <Button primary onClick={() => props.resetQuery()}>
+          "Clear query"
+        </Button>
+      </Segment>
+    </>
+  );
+};
+
+export const RDMErrorComponent = ({ error }) => {
+  return (
+    <Message warning>
+      <Message.Header>
+        <Icon name="warning sign" />
+        {error.response.data.message}
+      </Message.Header>
+    </Message>
+  );
+};
+
+export function SearchItemCreators({ creators }) {
+  function getIcon(creator) {
+    let ids = _get(creator, "person_or_org.identifiers", []);
+    let creatorName = _get(creator, "person_or_org.name", "No name");
+    let firstId = ids.filter((id) => ["orcid", "ror"].includes(id.scheme))[0];
+    firstId = firstId || { scheme: "" };
+    let icon = null;
+    switch (firstId.scheme) {
```

Comment:
for bigger discussion (not to address in this PR):
shouldn't these orcid and ror urls come from the backend?
ping @ntarocco @lnielsen

Reply:
Yes, they should be generated from the backend

Reply:
@Pineirin could you create a ticket for us not to forget to change it ? Please add it in our "Next sprint" column

Reply:
Ticket created: https://github.com/inveniosoftware/invenio-app-rdm/issues/1351